.eslintrc.js

```diff
@@ -26,5 +26,25 @@ module.exports = {
 		// do not throw an error if imported dependencies are 
 		// declared in `package.json` or `gutenberg/package.json`
 		"import/no-extraneous-dependencies": ["error", {"packageDir": ['.', './gutenberg/']}] 
-	}
+	},
+	overrides: [
+		{
+			// Workaround for addressing errors when importing react-native components.
+			// Related issue: https://git.io/JSKeJ
+			files: [ 'src/**/*.js' ],
+			settings: {
+				'import/ignore': [ 'react-native' ],
+			},
+		},
+		{
+			// Ignore test helpers import solving as the module is only available when using Jest.
+			files: [ 'src/**/test/*.js' ],
+			rules: {
+				'import/no-unresolved': [
+					'error',
+					{ ignore: [ 'test/helpers' ] },
+				],
+			},
+		},
```

Comment:
This rule has to be overridden for unit tests due to the fact that module "test/helpers" is mapped from Jest configuration ([reference](https://github.com/wordpress-mobile/gutenberg-mobile/blob/82227ef9da1e238b566e04a370e923f545cbc347/jest.config.js#L53)), so the ESLint resolver won't be able to solve it.

Reply:
Dropping a note that this relates to https://github.com/WordPress/gutenberg/pull/37073#discussion_r777730123. Why does this repository take a different approach of leveraging `overrides` in comparison to the inline ESLint comment found in Gutenberg?

Reply:
Right, in this case, I wanted only to ignore the path `test/helpers` but I agree that we should apply the same overriding as we do in Gutenberg 👍 .